.github/workflows/code-style.yml

```diff
@@ -21,22 +21,33 @@ jobs:
 
       - name: Check Out Code
         uses: actions/checkout@v4
-        with:
-          fetch-depth: 1
```

Comment:
Version 4 defaults to `fetch-depth: 1`

See https://github.com/actions/checkout?tab=readme-ov-file#usage